dependencies.yaml

```diff
@@ -245,32 +245,33 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - GPUtil>=1.4.0
-          - imagecodecs>=2021.6.8
-          - openslide-python>=1.1.2
           - psutil>=5.8.0
           - pytest-cov>=2.12.1
           - pytest-lazy-fixture>=0.6.3
           - pytest-xdist
           - pytest>=6.2.4
           - tifffile>=2022.7.28
+          - pooch>=1.6.0  # needed to download scikit-image sample data
       - output_types: [conda]
         packages:
+          - imagecodecs>=2021.6.8
```

Comment:
It seems like `imagecodecs` is pinned the same way for `conda` and `requirements`. Do you want to list that above in the `[conda, requirements, pyproject]` section?